Bottom: 407f6e7c934cd2b8cd3cc387090e26de0d321a32
Top:    22822e2857d2f84e2acfacba81a69e3c5da6ba36
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-05-24 21:11:04 -0400

Refresh of mpc5200-dma

---

diff --git a/sound/soc/fsl/mpc5200_dma.c b/sound/soc/fsl/mpc5200_dma.c
index a9ad54d..f6a9152 100644
--- a/sound/soc/fsl/mpc5200_dma.c
+++ b/sound/soc/fsl/mpc5200_dma.c
@@ -176,7 +176,8 @@ static int psc_dma_trigger(struct snd_pcm_substream *substream, int cmd)
 		 * end of stream and not over running it.
 		 */
 		s->runtime = runtime;
-		s->appl_ptr = s->runtime->control->appl_ptr - (runtime->period_size * runtime->periods);
+		s->appl_ptr = s->runtime->control->appl_ptr -
+				(runtime->period_size * runtime->periods);
 
 		/* Fill up the bestcomm bd queue and enable DMA.
 		 * This will begin filling the PSC's fifo.
@@ -327,9 +328,9 @@ static int
 psc_dma_hw_params(struct snd_pcm_substream *substream,
 			 struct snd_pcm_hw_params *params)
 {
-  	snd_pcm_set_runtime_buffer(substream, &substream->dma_buffer);
+	snd_pcm_set_runtime_buffer(substream, &substream->dma_buffer);
 
-  	return 0;
+	return 0;
 }
 
 static struct snd_pcm_ops psc_dma_ops = {
@@ -360,15 +361,15 @@ static int psc_dma_new(struct snd_card *card, struct snd_soc_dai *dai,
 		card->dev->coherent_dma_mask = 0xffffffff;
 
 	if (pcm->streams[0].substream) {
-		rc = snd_dma_alloc_pages(SNDRV_DMA_TYPE_DEV, pcm->card->dev, size,
-					&pcm->streams[0].substream->dma_buffer);
+		rc = snd_dma_alloc_pages(SNDRV_DMA_TYPE_DEV, pcm->card->dev,
+				size, &pcm->streams[0].substream->dma_buffer);
 		if (rc)
 			goto playback_alloc_err;
 	}
 
 	if (pcm->streams[1].substream) {
-		rc = snd_dma_alloc_pages(SNDRV_DMA_TYPE_DEV, pcm->card->dev, size,
-					&pcm->streams[1].substream->dma_buffer);
+		rc = snd_dma_alloc_pages(SNDRV_DMA_TYPE_DEV, pcm->card->dev,
+				size, &pcm->streams[1].substream->dma_buffer);
 		if (rc)
 			goto capture_alloc_err;
 	}
